UnityProject/Assets/Scripts/UI/Systems/Character/CharacterSettings.cs

```diff
@@ -79,6 +81,7 @@ public void ShowCharacterEditor(CharacterSheet character)
 			characterSelector.SetActive(false);
 			characterEditor.SetActive(true);
 			characterEditorScript.LoadCharacter(character);
+			UIManager.Instance.isInputFocus = true;
```

Comment:
Do you remember to unfocused it?

Reply:
it might be worth putting on  disable just to make sure

Reply:
pushed a change where it disables the focus when the entire window disappears